platform/mellanox/mlnx-onie-fw-update.sh

```diff
@@ -67,6 +67,13 @@ show_pending()
 	return $num
 }
 
+system_reboot()
+{
+	echo "Reboot will be done after 5 sec."
+	sleep 5
+	/sbin/reboot
```

Comment:
is it the same flow as we have for the sonic reboot? there is some work done in that reboot per vendor.

Reply:
@liat-grozovik no the flow is different. This need to be aligned with the SONiC reboot since it assumes power cycle rather then the regular reboot. I will fix it. Thanks.